cross-modal-search/app.py

```diff
@@ -28,44 +29,47 @@ def index_restful():
         flow.block()
 
 
-def check_index_result(resp):
-    for doc in resp.data.docs:
-        _doc = Document(doc)
-        print(f'{_doc.id[:10]}, buffer: {len(_doc.buffer)}, mime_type: {_doc.mime_type}, modality: {_doc.modality}, embed: {_doc.embedding.shape}, uri: {_doc.uri[:20]}')
-
-
 def check_query_result(resp):
     for doc in resp.data.docs:
         _doc = Document(doc)
         print(f'{_doc.id[:10]}, buffer: {len(_doc.buffer)}, embed: {_doc.embedding.shape}, uri: {_doc.uri[:20]}, chunks: {len(_doc.chunks)}, matches: {len(_doc.matches)}')
         if _doc.matches:
             for m in _doc.matches:
-                print(f'\t+- {m.id[:10]}, score: {m.scores["doc_score"].value}, text: {m.text}, modality: {m.modality}, uri: {m.uri[:20]}')
+                print(f'\t+- {m.id[:10]}, score: {m.scores["cosine"].value}, text: {m.text}, modality: {m.modality}, uri: {m.uri[:20]}, blob: {len(m.blob)}')
+                import matplotlib.pyplot as plt
```

Comment:
Since we are importing `matplotlib`, just wondering if we also need to add it to `requirement.txt`?